.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
why? what's this?

Reply:
sorry, i will delete it.
cause i read source codes of python-llfuse by pycharm from jetbrains, the ".idea" is its configuration directory of a project.